crates/hdk_derive/src/lib.rs

```diff
@@ -40,6 +43,26 @@ impl Parse for EntryDef {
                         }
                         _ => unreachable!(),
                     },
+                    "required_validation_package" => {
```

Comment:
Maybe just "validation_package" for the hdk specifier?

```suggestion
                    "validation_package" => {
```

Reply:
works for me